src/CommitmentsV2/SFA.DAS.CommitmentsV2/Data/Configuration/ApprenticeshipConfiguration.cs

```diff
@@ -53,7 +53,7 @@ public void Configure(EntityTypeBuilder<Apprenticeship> builder)
             builder.HasOne(d => d.Cohort)
                 .WithMany(p => p.Apprenticeships)
                 .HasForeignKey(d => d.CommitmentId)
-                .OnDelete(DeleteBehavior.ClientSetNull);
+                .OnDelete(DeleteBehavior.Cascade);
```

Comment:
As a matter of interest what related tables will have entries deleted? ClientSetNull does seem wrong as well (at least to me)

Reply:
It deletes from the Apprenticeship table.
With ClientSetNull, it was trying to set null in Apprenticeship.CommitmentID column.